Tests/TuistLoaderTests/Loaders/ManifestModelConverterTests.swift

```diff
@@ -308,14 +310,49 @@ class ManifestModelConverterTests: TuistUnitTestCase {
         XCTAssertEqual(model.projects, [])
     }
 
-    // MARK: - Helpers
+    func test_loadDependenciesGraph_withExternalSPM() throws {
+        // Given
+        let temporaryPath = try temporaryPath()
+        try fileHandler.createFolder(temporaryPath.appending(components: ["checkouts", "Alamofire", "Source"]))
+        let manifest = DependenciesGraphManifest.alamofire(spmFolder: .path(temporaryPath.pathString))
+        let subject = makeSubject(with: makeManifestLoader())
+
+        // When
+        let model = try subject.convert(manifest: manifest, path: temporaryPath)
+
+        // Then
+        XCTAssertEqual(model.externalProjects.values.first?.isExternal, true)
+    }
+
+    func test_loadDependenciesGraph_withLocalSPM() throws {
+        // Given
+        let temporaryPath = try temporaryPath()
+        try fileHandler.createFolder(temporaryPath.appending(components: ["ADependency", "Sources", "ALibrary"]))
+        try fileHandler.createFolder(temporaryPath.appending(components: ["ADependency", "Sources", "ALibraryUtils"]))
+        let manifest = DependenciesGraphManifest.aDependency(spmFolder: .path(temporaryPath.pathString))
+        let subject = makeSubject(with: makeManifestLoader())
 
+        // When
+        let model = try subject.convert(manifest: manifest, path: temporaryPath)
+
+        // Then
+        XCTAssertEqual(model.externalProjects.values.first?.isExternal, false)
+    }
+}
+
+// MARK: - Helpers
+
+extension ManifestModelConverterTests {
     func makeSubject(with manifestLoader: ManifestLoading) -> ManifestModelConverter {
         ManifestModelConverter(
             manifestLoader: manifestLoader
         )
     }
 
+    func makeManifestLoader() -> ManifestLoading {
```

Comment:
You can specify a default parameter [here](https://github.com/tuist/tuist/pull/6436/files#diff-ee62605e4bc3ae64f0d86d9479bfc68aefc22d336a7916e0b1c1d495f16b31ecR356) instead of defining the method again.